### PR TITLE
Upgrade to SBT 0.13.5 and make it an AutoPlugin

### PR DESCRIPTION
**Pull request by benmccann**
A version of https://github.com/typesafehub/sbteclipse/pull/211 that can be automatically merged

### CODE REVIEW

**Comment by er1c** [comment]
Should we change out: https://github.com/typesafehub/sbteclipse/blob/master/project/Build.scala#L54 too?

**Comment by er1c** [comment]
This looks to be @jroper  (https://github.com/typesafehub/sbteclipse/commit/9ae5504ef6ce7d40d64b94c9a69e308345643feb) and @hseeberger (https://github.com/typesafehub/sbteclipse/commit/f1cdfce2ac894b457af982536f250a69127d13b2) call if they know that part of the code best :)

File: project/Build.scala [hunk 1]
```diff
@@ -41,27 +41,25 @@ object Build extends Build {
       organization := "com.typesafe.sbteclipse",
       // version is defined in version.sbt in order to support sbt-release
       scalacOptions ++= Seq("-unchecked", "-deprecation"),
-      publishTo <<= isSnapshot { isSnapshot =>
-        val id = if (isSnapshot) "snapshots" else "releases"
+      publishTo := {
+        val id = if (isSnapshot.value) "snapshots" else "releases"
```
Comment (jsuereth) [inline project/Build.scala]: this feels so good to see....

**Comment by jsuereth** [comment]
LGTM

**Pull request merged**